packages/react-shortcuts/README.md

```diff
@@ -0,0 +1,156 @@
+# `@shopify/react-shortcuts`
+
+![bundle size badge](https://img.shields.io/bundlephobia/minzip/@shopify/react-shortcuts.svg)
```

Comment:
Adding this badge to our libraries lets developers easily understand the cost of adding our library to their project.

Reply:
Yeah I think having both in both places is the best. Keep in mind a lot of people will probably get to these readmes from the NPM link, which makes it higher value to have them in the README for the package itself.